bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/handler/DeviceHandler.java

```diff
@@ -140,10 +140,11 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                                 thing.getChannel(channelUID.getId()));
                     } else if (command instanceof QuantityType<?>) {
                         QuantityType<?> value = (QuantityType<?>) command;
-                        Integer f = value.intValue();
+                        Double f = value.doubleValue();
```

Comment:
```suggestion
                        double f = value.doubleValue();
```

Use primitives where possible